.github/workflows/ci.yml

```diff
@@ -15,42 +15,65 @@ jobs:
   org-check:
     name: Check GitHub Organization
     if: github.repository_owner == 'a-scie'
-    runs-on: ubuntu-22.04
+    runs-on: ubuntu-24.04
     steps:
       - name: Noop
         run: "true"
   ci:
-    name: (${{ matrix.os }}) CI
+    name: ${{ matrix.name }} CI
     needs: org-check
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
         # N.B.: macos-13 is the oldest non-deprecated Intel Mac runner and macos-14 is the oldest
         # non-deprecated ARM Mac runner.
-        os: [ ubuntu-22.04, linux-arm64, macos-13, macos-14, windows-2022, windows-arm64 ]
+        include:
+          - os: ubuntu-24.04
+            name: Linux x86-64
+          - os: ubuntu-24.04
+            name: Linux x86-64 (musl)
+            cross-target: x86_64-unknown-linux-musl
+          - os: ubuntu-24.04
+            name: Linux aarch64
+            cross-target: aarch64-unknown-linux-musl
+          - os: ubuntu-24.04
+            name: Linux armv7l
+            cross-target: armv7-unknown-linux-musleabihf
```

Comment:
@Lauszus note that no cargo config is needed for linker flags, just specifying the musl targets is enough to get you a statically linked executable:
```
:; file dist/ptex-linux-* | grep -v sha256
dist/ptex-linux-aarch64:        ELF 64-bit LSB executable, ARM aarch64, version 1 (SYSV), statically linked, stripped
dist/ptex-linux-arm:            ELF 32-bit LSB executable, ARM, EABI5 version 1 (SYSV), statically linked, stripped
dist/ptex-linux-x86_64:         ELF 64-bit LSB pie executable, x86-64, version 1 (SYSV), static-pie linked, stripped
```

I built those on my GNU-Linux x86-64 box using `cross run -p package --target {x86_64-unknown-linux-musl,aarch64-unknown-linux-musl,armv7-unknown-linux-musleabihf}`.